src/App.jsx

```diff
@@ -1,28 +1,91 @@
 import React from 'react';
 import './App.scss';
 import './styles/general.scss';
+import { getTodos } from './api';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
 
+
 class App extends React.Component {
   state = {
     todos: [],
+    prepearedTodos: [],
     selectedUserId: 0,
   };
 
+  componentDidMount() {
+    getTodos()
+      .then(todos => this.setState({
+        todos: todos.filter(todo => todo.title),
+        prepearedTodos: todos,
+      }));
+  }
+
+  nameFilter = (input) => {
+    this.setState(state => ({
+      todos: [...state.prepearedTodos].filter(todo => (todo.title
```

Comment:
```suggestion
      todos: state.prepearedTodos.filter(todo => (todo.title
```
filter returns a copy of the array